prefect_snowflake/credentials.py

```diff
@@ -85,7 +85,7 @@ class SnowflakeCredentials(CredentialsBlock):
     password: Optional[SecretStr] = Field(
         default=None, description="The password used to authenticate."
     )
-    private_key: Optional[SecretBytes] = Field(
+    private_key: Optional[SecretStr] = Field(
```

Comment:
Is the error you see resolved if this field accepts either a `SecretStr` or `SecretBytes`?
```suggestion
    private_key: Optional[Union[SecretStr, SecretBytes]] = Field(
```